### PR TITLE
hMail Anti-spam

### PR DESCRIPTION
**Pull request by CountBacula**
* hMail Anti-spam

Created new category for hMail and doc on hMail anti-spam configuration. Submitted on behalf of Nathan.

### CODE REVIEW

File: source/operatingsystems/windows/hmail/antispamsetup.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# hMailServer - Anti-spam Setup
+
+hMailServer comes with some Anti-Spam protection enabled out-of-the-box. This article is designed to help you make sure this is configured and proviide some further options to help make it its spam detection even better.
```
Comment (terencesimms) [inline source/operatingsystems/windows/hmail/antispamsetup.md]: *provide

File: source/operatingsystems/windows/hmail/antispamsetup.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+## Spam Tests
+
+![hMailServer > Settings > Anti-spam > Spam Tests](files/antispam/spamtests.png)
+
+- **Use SPF** - Default value is "3". This will only trigger when a domain hard-fails an SPF check. As such if you are seeing issues with spam, we would recommend changing this to 5. You can read further about SPF in our [main SPF guide](https://docs.ukfast.co.uk/operatingsystems/linux/mail/spf.html).
+
+- **Check host in the HELO Command** - Default value is "2". This will compare the hostname used in the sending mail server's HELO to its rDNS (PTR) record. We would recommend changing this to "3".
+
+- **Check that sender had DNS-MX records** - Default value is "2". This will make sure that the sender domain has MX reccordds. Any legitimate domain sending mail should have MX records. As such, we recommend setting this score to "5".
```
Comment (terencesimms) [inline source/operatingsystems/windows/hmail/antispamsetup.md]: *records

File: source/operatingsystems/windows/hmail/antispamsetup.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+![hMailServer > Settings > Anti-spam > SURBL servers](files/antispam/surbl.png)
+
+**SURBL servers** work differently to DNS check lists by checking links in the body of the message. This will check incoming emails to see if they contain links to known-malicious websites. 
+
+## Grey Listing
+
+![hMailServer > Settings > Anti-spam > Greylisting](files/antispam/greylisting.png)
+
+**Grey Listing** works by _temporarily_ rejecting any mail sent to your server - a correctly configured mail server will almost always attempt to resend a message rejected in this manner, being successful on the second attempt.
```
Comment (terencesimms) [inline source/operatingsystems/windows/hmail/antispamsetup.md]: Perhaps ' ... to your server. A correctly configured..."
or a semi-colon? '...to your server; a correctly configured..."

or just leave be.

File: source/operatingsystems/windows/hmail/antispamsetup.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+**SURBL servers** work differently to DNS check lists by checking links in the body of the message. This will check incoming emails to see if they contain links to known-malicious websites. 
+
+## Grey Listing
+
+![hMailServer > Settings > Anti-spam > Greylisting](files/antispam/greylisting.png)
+
+**Grey Listing** works by _temporarily_ rejecting any mail sent to your server - a correctly configured mail server will almost always attempt to resend a message rejected in this manner, being successful on the second attempt. 
+
+hMail will keep a log of the sending IP for the initial message and compare this against any re-try attempts. Most spam mail servers will typically _not_ try to resend a message once it has been rejected.
```
Comment (terencesimms) [inline source/operatingsystems/windows/hmail/antispamsetup.md]: *retry

File: source/operatingsystems/windows/hmail/antispamsetup.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+## Grey Listing
+
+![hMailServer > Settings > Anti-spam > Greylisting](files/antispam/greylisting.png)
+
+**Grey Listing** works by _temporarily_ rejecting any mail sent to your server - a correctly configured mail server will almost always attempt to resend a message rejected in this manner, being successful on the second attempt. 
+
+hMail will keep a log of the sending IP for the initial message and compare this against any re-try attempts. Most spam mail servers will typically _not_ try to resend a message once it has been rejected. 
+
+This process can cause issues as the re-try delay can be lengthy (minutes / hours) and large mail servers can send from multiple IPs. This can cause delays in delivery to your server until the same email is sent by the same IP twice. For this reason, we typically advise against enabling Grey Listing.
```
Comment (terencesimms) [inline source/operatingsystems/windows/hmail/antispamsetup.md]: retry

**Review by terencesimms — COMMENTED** [review]
Just some tiny typos. All good otherwise.

File: source/operatingsystems/windows/hmail/antispamsetup.md (hunk 3 above)
Comment (CountBacula) [reply source/operatingsystems/windows/hmail/antispamsetup.md]: semi-colon it is

**Review by terencesimms — APPROVED** [review]

**Review by martynyih — APPROVED** [review]

**Pull request merged**